text/0100-sensitive-data-handling.md

```diff
@@ -0,0 +1,103 @@
+#  Sensitive Data Handling
+
+By default, OpenTelemetry libraries never capture potentially-sensitive data, except for the full URL.
```

Comment:
That's a pretty bold statement. Are we sure we will be able to ensure this 100%?

Reply:
It's a design goal.  Bugs are probably inevitable 😄

Reply:
I think armin was asking if this is a design goal we want. The term `libraries` could mean anything from API to SDK to Instrumentation Libraries. I think the strongest statement we could make would be:
```suggestion
By default, instrumentation libraries provided by OpenTelemetry make a best effort to never capture potentially-sensitive data, except in cases like the full URL where the data is necessary to the observability of the system.
```